WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -820,9 +820,13 @@ public static void scrollIntoView(Integer scrollableContainerID, ViewInteraction
     }
 
     public static void dismissJetpackAdIfPresent() {
+        String jpAdText = "Stats, Reader, Notifications, and other features are powered by Jetpack.";
+
         // Dismiss Jetpack ad that might be shown after Sign-Up or after opening Stats
-        if (isElementDisplayed(onView(withText("Jetpack powered")))) {
+        if (isElementDisplayed(onView(withText(jpAdText)))) {
             clickOn(onView(withId(R.id.secondary_button)));
         }
+
+        idleFor(3000);
```

Comment:
> Also when this is not added, do you see the test failing?

Yes, this is the reason why this was added. However, I'm not sure that the test failed exactly because of the banner being still partly shown. Anyhow, it seemed correct to me of give banner a time to disappear.

> I'm a little wary of adding implicit waits to tests.

Fully agree. Honestly, I think I was too desperate to have this PR merged as fast as possible. I've improved the situation in - 1905e161316c7cfbe146e4fd70f3b287e9c39d72 - please take a look.

Thank you 🙇

Reply:
Adding my 2 cents here, just in case you are not aware of that, and I am sure you are, but one can't be too careful, can they... 😅 So, instead of adding sleep the recommending approach is to using [idling resources](https://developer.android.com/training/testing/espresso/idling-resource) instead. However, I do understand that doing so is not as straightforward and sometimes require some development. So, please use it when you see it being fit, when you feel that sleep is being abused just because it's easy to add and you think it can be avoided like that.

Reply:
@ParaskP7 👋 

> ... but one can't be too careful, can they...😅

So true, because I never used idling resources before, so thank you for mentioning this, it looks like something at least very worth knowing! I still feel quite new to Espresso.

Reply:
Oh yes, I really recommend looking into [idling resources](https://developer.android.com/training/testing/espresso/idling-resource), it is really a game changer in terms of making tests less flaky and removing unnecessary sleeping for no apparent reason! 💯

Reply:
Thanks for sharing about idling resources @ParaskP7 ! I've not used it before (and I don't think there's an equivalent in other UI test frameworks I've worked with before...) Looks very interesting, I'll definitely check it out

Reply:
Great, I am so excited for you both, I think you will like it! 💯 

PS: If you need any help with it, don't be shy, just let me know and I'll try to pair with you in helping you with anything I can [idling resources](https://developer.android.com/training/testing/espresso/idling-resource) related!